metals-docs/src/test/scala/docs/SnapshotTest.scala

```diff
@@ -0,0 +1,31 @@
+package docs
+
+import java.time._
+import utest._
+
+object SnapshotTest extends TestSuite {
```

Comment:
The `mtest` project in this build contains a `BaseSuite` that most of our test suites extends and exposes a `test(name: String): Unit` method. Most commonly, we structure test suites in this repo like here https://github.com/scalameta/metals/blob/6693914e4662220df4a04ce3ab841a11a851c0b0/tests/unit/src/test/scala/tests/TimerSuite.scala by creating a `def check(name: String, original: String, expected: ZonedDateTime): Unit` that is called like this
```scala
check(
  "basic",
  "Mon Apr 08 04:09:49 UTC 2019",
  ZonedDateTime.of(2019, 4, 8, 4, 9, 49, 0, ZoneId.of("UTC"))
)
// ...
```